### PR TITLE
Add Latest News to Admin 

### PR DESCRIPTION
**Pull request by evelynhasama**
## Description
Instead of having hard coded information for the "Latest From ASPC" section of the home page, we want to allow admins to add their own info through the Admin portal. Now, in the Admin portal under "Latest News", admins can create/edit/delete new Latest News Items that contain a title, caption, URL, and priority.

**NOTE:** URLs must have https or link to pomonastudents.org/...

In the backend, we added a new table `LatestNews` to db to create and save latest news items.  
Notes on how to create new table and update schema with rails for future:

In docker rails container (exec) run:

```
bin/rails generate model LatestNews title:string caption:string details_url:string priority:integer
bin/rails db:migrate
```

On the frontend, these latest news items are rendered on the right column of the home page, where they have clickable links.  Only ** top 2 priority items** are rendered on the home screen to keep the screen structure in balance.

## Test
- tested locally, creating, deleting, editing latest news

### Screenshots

**Latest News in Admin Dashboard** 
dashboard

**Rendered under Latest From ASPC**
latest_from_aspc
note: only top 2 priority items displayed; if a link is not present, button is not shown

### CODE REVIEW

File: app/models/latest_news.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class LatestNews < ApplicationRecord
```
Comment (chauvuha) [inline app/models/latest_news.rb:1]: Add/delete/update all work great! One comment I have is maybe we can make the 'title' and 'priority' fields required, to help admins avoid adding empty/ news items without an order or keep track of the order they should be displayed in with these two lines:

```
validates :title, presence: true
validates :priority, presence: true
```

**Review by chauvuha — CHANGES_REQUESTED** [review]
Looks great! I only have one suggestion on making priority & title fields not empty

File: app/models/latest_news.rb (hunk 1 above)
Comment (evelynhasama) [reply app/models/latest_news.rb:1]: yep good idea!

**Review by chauvuha — APPROVED** [review]
Looks good!

**Pull request merged**